core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -126,9 +130,11 @@ public void init() throws StoreInitializationException {
     List<Broker> brokers = JavaConversions.seqAsJavaList(brokerSeq);
     String bootstrapBrokers = "";
     for (int i = 0; i < brokers.size(); i++) {
-      bootstrapBrokers += brokers.get(i).connectionString();
-      if (i != (brokers.size() - 1)) {
-        bootstrapBrokers += ",";
+      for(EndPoint ep : JavaConversions.asJavaCollection(brokers.get(i).endPoints().values())) {
+        bootstrapBrokers += ep.connectionString();
+        if (i != (brokers.size() - 1)) {
+          bootstrapBrokers += ",";
```

Comment:
Won't this break if the last broker has multiple endpoints? (I.e. if we're at the last broker, no comma would be added to separate endpoints)